packages/opentelemetry-api/src/metrics/ObserverResult.ts

```diff
@@ -15,11 +15,11 @@
  */
 
 import { Labels } from './Metric';
+import { MetricObservable } from './MetricObservable';
 
 /**
  * Interface that is being used in function setCallback for Observer Metric
  */
 export interface ObserverResult {
-  observers: Map<Labels, Function>;
```

Comment:
This is a breaking change if you remove this map. Is that intentional?

Reply:
yes it is intentional as it should not be there from the very beginning - it  was rather bug